docs/reference/migration/migrate_8_0.asciidoc

```diff
@@ -12,6 +12,7 @@ See also <<release-highlights>> and <<es-release-notes>>.
 coming[8.0.0]
 
 * <<breaking_80_analysis_changes>>
+* <<breaking_80_breaker_changes>>
```

Comment:
There was a test failure on the [docs build](https://elasticsearch-ci.elastic.co/job/elastic+docs+elasticsearch+pull-request/152/console):

```
19:31:27 INFO:build_docs:/tmp/docsbuild/target_repo/html/en/elasticsearch/reference/master/index.x.xml:65535: element xref: validity error : IDREF attribute linkend references an unknown ID "breaking_80_breaker_changes"
```
I think this is caused by a missing include of the `breaker.asciidoc` file towards the end of this file.

Can you please double-check that all CI checks are green when you resubmit?